### DISCUSSION TITLE
AutoJoomer Beta

### DISCUSSION THREAD

**Discussion by vishal-lokare** [vishal-lokare/AutoJoomer#115] [Ideas]
AutoJoomer Beta 2.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/vishal-lokare/AutoJoomer/releases/tag/'>AutoJoomer Beta</a>.</em>

**Comment by joshi008** [comment]
Need for an active inactive toggle bar in the extension. Since 2019 batch is having their exams, and in the middle of the exam the extension pops up to notify and join the session scheduled.

**Reply by vishal-lokare** [reply]
Noted, we have it planned already will be added soon
